apps/andi/assets/css/app.scss

```diff
@@ -12,7 +12,7 @@ body {
   flex-direction: column;
 
   font-family: Arial, Helvetica, sans-serif;
-  font-size: 0.75em;
+  font-size: 1em;
```

Comment:
Rems vs ems?

Reply:
I think we normed on moving to rems over ems, but nobody actually made the change yet.  Do y'all mind managing that find/replace?

Reply:
no problemo